### DISCUSSION TITLE
Facing error on Windows 11 Insider Preview 25151.1000 (rs_prerelease) build

### DISCUSSION THREAD

**Discussion by samiran9200** [AveYo/MediaCreationTool.bat#165] [Q&A]
Facing error on Windows 11 Insider Preview 25151.1000 (rs_prerelease) build
Install error - 0xc1900130
Skip_TPM_Check_on_Dynamic_Update is not working correctly on this build unable to update windows
Please fix it

**Comment by AveYo** [comment]
that is a setup error, **after** the bypass part - script has done it's job
plus it is verified working by many users including myself
![Imgur](https://i.imgur.com/ZtiQNYY.png)

might want to try again later in case of network congestion / peering issue
try the windows update refresh script from here
try troubleshooting steps you can find online for the error code
zip and share `C:\$WINDOWS.~BT\Sources\Panther\` and I'll check what happens on your end (when I have some time, this is beyond the scope of supporting the script)
last but not least, Dev builds are not guaranteed to install, and neither is windows update in general

**Reply by AveYo** [reply]
![Imgur](https://i.imgur.com/gBqWeNd.png)

**Comment by samiran9200** [comment]
Tried to troubleshoot but not working please help me
[Panther.zip](https://github.com/AveYo/MediaCreationTool.bat/files/9024865/Panther.zip)

**Comment by AveYo** [comment]
Setup_InsufficientSystemPartitionDiskSpace

2022-07-01 08:39:09, Info CFreeSystemPartitionDiskSpaceChecker: 
checked InsufficientSystemPartitionDiskSpace, found HardBlock.
Not enough space on partition [\\?\GLOBALROOT\Device\HarddiskVolume1\]
Required: 15 MB. Available: 5 MB.

**That's the EFI system partition!**

You could try (if you understand the risks of making your system unbootable):
`diskpart`
`list disk`
pick the one where os is installed, probably 0, and enter below
`select disk 0`
`list part`
pick the system one around 100MB, probably 1, and enter below
`select part 1`
`assign`
`exit`
now a new drive letter should appear in explorer, open that
in `\EFI\Microsoft\Boot\Fonts` delete `cht_boot.ttf` `chs_boot.ttf` `kor_boot.ttf` `jpn_boot.ttf`
in `\EFI\Microsoft\Boot\` delete the many country-code folders except `en-us` and `Fonts` `Resources` `qps-ploc`
should be fine as your os is en-us, and that should free more than setup needs
try again
the root issue might be with your BCD logs being bloated but it's something the upgrade should sort out

**Comment by fana13** [comment]
Hi @AveYo 
I've got a similar issue with basic 22h2 update or now with insider preview 10.0.22623.730 (ni_release), i got the 0xc1900130
I tried a loooot of solutions and nothing works.
With your tool i've just got an error like "windows 11 installation failed" and OK.

Here the C:\$WINDOWS.~BT\Sources\Panther\, if you have some times to save my life ^^
Thanks.

[Panther.zip](https://github.com/AveYo/MediaCreationTool.bat/files/9726315/Panther.zip)

**Comment by samiran9200** [comment]
Thanks sir. Working now
